usr/share/rear/lib/global-functions.sh

```diff
@@ -503,7 +503,7 @@ function mount_url() {
 
     # The cases where we return 0 are those that do not need umount and also do not need ExitTask handling.
     # They thus need to be kept in sync with umount_url() so that RemoveExitTasks is used
-    # iff AddExitTask was used in mount_url().
+    # if AddExitTask was used in mount_url().
```

Comment:
Why this change? I suppose the `iff` was intentional.

Reply:
I think
```
git log -p --follow usr/share/rear/lib/global-functions.sh
```
proves that both `iff` are intentional
because they were introduced via
https://github.com/rear/rear/commit/1ea884cfa378328ac2f7d5e1ab507dded78fd3db
and because `"if" != "iff"` such changes
change the (logical) meaning of the comment.